src/Microsoft.Android.Sdk.ILLink/TypeMappingStep.cs

```diff
@@ -135,4 +261,27 @@ void ProcessType (AssemblyDefinition assembly, TypeDefinition type)
 		foreach (TypeDefinition nested in type.NestedTypes)
 			ProcessType (assembly, nested);
 	}
+
+	ulong Hash (string javaName)
+	{
+		ReadOnlySpan<byte> bytes = MemoryMarshal.AsBytes(javaName.AsSpan ());
+		return _hashMethod!(bytes);
+	}
+
+	static HashMethod GetHashMethod (AssemblyLoadContext alc, string assemblyPath)
+	{
+		System.Reflection.Assembly? hashingAssembly;
+		try {
+			hashingAssembly = alc.LoadFromAssemblyPath (assemblyPath);
+		} catch (Exception ex) {
+			throw new InvalidOperationException ($"Unable to load {assemblyPath}, {nameof(TypeMappingStep)} cannot proceed", ex);
+		}
+
+		var hashToUInt64 = hashingAssembly?.GetType ("System.IO.Hashing.XxHash3")?.GetMethod ("HashToUInt64");
```

Comment:
```suggestion
		System.Reflection.Assembly hashingAssembly;
		try {
			hashingAssembly = alc.LoadFromAssemblyPath (assemblyPath);
		} catch (Exception ex) {
			throw new InvalidOperationException ($"Unable to load {assemblyPath}, {nameof(TypeMappingStep)} cannot proceed", ex);
		}

		var hashToUInt64 = hashingAssembly.GetType ("System.IO.Hashing.XxHash3").GetMethod ("HashToUInt64");
```

The `LoadFromAssemblyPath()` API returns a non-nullable `Assembly`.